examples/multimodal/load.ts

```diff
@@ -25,12 +27,16 @@ async function generateDatasource() {
     const documents = await new SimpleDirectoryReader().loadData({
       directoryPath: path.join("multimodal", "data"),
     });
-    const storageContext = await storageContextFromDefaults({
-      persistDir: "storage",
-      storeImages: true,
+    const clipVectorStore = new SimpleVectorStore({
+      embedModel: new ClipEmbedding(),
```

Comment:
@himself65 `pnpm type-check` produces here:
```
Object literal may only specify known properties, and 'embedModel' does not exist in type '{ data?: SimpleVectorStoreData | undefined; }'.
```
doesn't make a lot of sense to me as `SimpleVectorStore` is produced by a mixin and the mixin's constructor allows any

Reply:
I added some changes, I think type here is indeed wrong